README.md

```diff
@@ -229,6 +229,7 @@ Alternatively, install Starship using any of the following package managers:
 | Gentoo             | [Gentoo Packages]       | `emerge app-shells/starship`                                  |
 | Manjaro            |                         | `pacman -S starship`                                          |
 | NixOS              | [nixpkgs]               | `nix-env -iA nixpkgs.starship`                                |
+| openSUSE           | [openSUSE Tumbleweed]   | `zypper install starship`                                     |
```

Comment:
You also need to include a
```
[openSUSE Tumbleweed]: https://some.url.here
```

At the end of the file to make this a link.

Reply:
Furthermore, if repology is to be trusted, Leap also has starship packages.